### DISCUSSION TITLE
Commercialization

### DISCUSSION THREAD

**Discussion by mikeabraham** [refinedev/refine#3261] [General]
This is a terrific product and you have made tremendous strides in just a short time. This question is directed at Refine the company, rather than to the full audience of contributors.

In my opinion, there is a growing trend toward a split, open source versus commercial, distribution model. Some companies open source the code but offer commercial hosting, while others open source certain features but close source others. Most of the competitors in the 'admin dashboard' space seem to do one or the other or both. 

My question is, what is your plan, if you're willing and able to share it?

Speaking only for myself, it's hard to lock in to a codebase that requires a monthly or annual subscription, and knowing that future was inevitable for Refine might influence my decision to use it. There's nothing wrong with a subscription model per se, but cost can be a factor in making technology decisions.

Obviously, the contributors to Refine (and any other open source product) need to make a living and have every right to pursue whatever commercialization of the product they desire. I support whatever avenue you choose wholeheartedly. If you are able to share your plans, I would be interested in hearing them.

And again, great work.

**Comment by drcivan** [comment]
Thank you for your kind and motivating words about the project. We always appreciate positive feedback as it helps us know we're doing our best for the community. 🙏

Your question about the commercialization path of refine is valid and I understand that it is one of the critical factors for the decision makers by selecting the framework for large scale projects. 

Although we are quite a few steps away from monetization and have our primary focus on extending the user base, I can still share our fundamental principles with the community:

refine has adopted an "open core" model, and the current open-source codebase offers developers a generous set of features that are unmatched by alternative solutions. 

We'll stick to this approach in the future and keep the core product forever free. We pledge the community that we have no intention of moving any of the current features behind a paywall.

Any commercial, close-sourced products or features we'll add to refine ecosystem in the future, will be rather complimentary to the core product. These solutions will be offered on an opt-in basis for developers.

I hope this answer has clarified any concerns you or other developers may have had regarding the commercialization issue.

Best,
